### PR TITLE
cifar10

### PR DESCRIPTION
**Pull request by tanimutomo**
Check List
- [x] Model
- [x] Loss
- [x] Dataset
- [x] Augmentation
- [x] Hyper paramerters
- [x] weight init

### CODE REVIEW

**Comment by tanimutomo** [comment]
datasetとtransformは，pytorch実装だったので，そのまま利用する

**Comment by tanimutomo** [comment]
weight init is 'he'

**Comment by tanimutomo** [comment]
hparamsはscrapboxにまとめた

**Comment by tanimutomo** [comment]
- CIFAR10のためにResNetのModelを追加 & 修正
- parserをclassにしてupdateした，default_optionsを追加して，datasetに応じて無指定のoptionを書き換えるようにした

**Comment by tanimutomo** [comment]
- 無駄なoptionを排除
- data parallel
- test scriptを追加
- option等もtestに合わせて改変

**Comment by tanimutomo** [comment]
AEsを使う学習が全てか学習していたので，ct_atだけ，weight decayを追加して再度学習中．
